__tests__/app.test.js

```diff
@@ -65,10 +65,42 @@ describe('Getting a list of available api', ()=> {
   })
 
 
+  describe('test to see if input id of article exists', ()=> {
```

Comment:
We'd usualy use describe blocks to group together tests for an endpoint, our test descriptions are then used to describe the behaviour we're expecng from that test.